src/plugin.js

```diff
@@ -351,6 +352,10 @@ const eme = function(options = {}) {
         }
       }
     },
+    // Pass a promise polyfill from the player options for IE support. If none
+    // exists, native Promises will be used and the function won't be supported in IE
+    detectSupportedCDMs: createDetectSupportedCDMsFunc(player.options().Promise),
+    getSupportedCDMs,
```

Comment:
why have a synchronous function for an asynchronous process?

Reply:
Could just get rid of it. Curious whether others think it makes sense to remove `getSupportedCDMs()`

Reply:
I think getting rid of `getSupportedCDMs` is the right call here.